src/Symfony/Component/Validator/Constraints/Range.php

```diff
@@ -25,18 +25,22 @@ class Range extends Constraint
     const INVALID_CHARACTERS_ERROR = 'ad9a9798-7a99-4df7-8ce9-46e416a1e60b';
     const TOO_HIGH_ERROR = '2d28afcb-e32e-45fb-a815-01c431a86a69';
     const TOO_LOW_ERROR = '76454e69-502c-46c5-9643-f447d837c4d5';
+    const INVALID_STEP_ERROR = 'de5ca5b7-8fcf-49cf-97f7-1319b4fe8e5e';
```

Comment:
`public const`?

Reply:
isn't it public by default ?

Reply:
Yes, but I think we should be explicit now that we can do that for constants (just like we deal with class properties).